README.md

```diff
@@ -52,4 +54,3 @@ Specifically you can:
   - Alireza Mahmoudian ([BelegCuthalion](https://github.com/BelegCuthalion))
   - Amirhossein Yaghoobi from Iran ([amir-yaghoobi](https://github.com/amir-yaghoobi))
   - Hussein Habibi Juybari from Iran 🇮🇷([husseinhj](https://github.com/Husseinhj))
-  - Mahdi Ghiasi from Iran ([MahdiGhiasi](https://github.com/MahdiGhiasi))
```

Comment:
Ooh! I didn't do this maybe after I clone this repo this line added

Reply:
I fixed it.
Add your name to supporters if you want